### PR TITLE
Resize and NoMove updates on redaction box

### PR DESCRIPTION
**Pull request by aparna-aot**
Redactions moved from one page to other - breaks grouping and other issues#4311
Dragging full page redaction box#4399

### CODE REVIEW

File: web/src/components/FOI/Home/Redlining.js [hunk 1]
```diff
@@ -1437,6 +1448,7 @@ const Redlining = React.forwardRef(
           });
         }
       } else {
+        console.log("_resizeAnnot!!!:",_resizeAnnot);
```
Comment (nkan-aot) [inline web/src/components/FOI/Home/Redlining.js:1451]: remove

File: web/src/components/FOI/Home/Redlining.js [hunk 2]
```diff
@@ -780,6 +780,8 @@ const Redlining = React.forwardRef(
                     ];
                   individualPageNo = displayedDoc.page;
                   if (annotations[i]?.type === "fullPage") {
+                    annotations[i].NoResize = true;
+                    annotations[i].NoMove = true;
```
Comment (nkan-aot) [inline web/src/components/FOI/Home/Redlining.js:784]: is line 784 still neccessary?

**Review by nkan-aot — APPROVED** [review]

File: web/src/components/FOI/Home/Redlining.js (hunk 1 above)
Comment (aparna-aot) [reply web/src/components/FOI/Home/Redlining.js:1451]: Already removed with another commit.

File: web/src/components/FOI/Home/Redlining.js (hunk 2 above)
Comment (aparna-aot) [reply web/src/components/FOI/Home/Redlining.js:784]: NoResize is required as its only for fullpage redactions . Removed NoMove .

**Pull request merged**